source/common/stats/thread_local_store.cc

```diff
@@ -106,6 +109,18 @@ void ThreadLocalStoreImpl::shutdownThreading() {
   shutting_down_ = true;
 }
 
+void ThreadLocalStoreImpl::RegisterMutexContentionGauge() {
+  mutex_contention_instances = &default_scope_->counter("mutex.contention.instances");
+  mutex_contention_wait_cycles = &default_scope_->gauge("mutex.contention.wait_cycles");
+
+  absl::RegisterMutexTracer([](const char* msg, const void* obj, int64_t wait_cycles) -> void {
```

Comment:
imo  we should not call this more than once per process eg in tests.

Reply:
Flushing comments as out of date.